marklogic-data-hub-central/ui/src/components/modal-status/modal-status.test.tsx

```diff
@@ -1,12 +1,211 @@
+        error : { 
+          title: '', 
+          message: '',
+          type: ''
+        },
+        maxSessionTime: 300,
+        sessionWarning: true
+      },
+      userNotAuthenticated: jest.fn(),
+      handleError: jest.fn(),
+      resetSessionTime: jest.fn(),
+      setSessionWarning: jest.fn(),
+      clearErrorMessage: jest.fn()
+    }
+    const { getByText } = render( 
+      <Router>
+        <UserContext.Provider value={context}>
+          <ModalStatus/>
+        </UserContext.Provider>
+      </Router>);
+
+      expect(getByText('Due to Inactivity, you will be logged out in')).toBeInTheDocument();
+
+      await wait(() => {
+        userEvent.click(getByText('Log Out'));
+      });
+      expect(axiosMock.get).toHaveBeenCalledTimes(1);
+  });
+
+  test('Modal can render 500 error and can click OK', async () => {
+    const context = {
+      user: {
+        error : { 
+          title: '500 Internal Server Error', 
+          message: 'java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:8011',
+          type: 'MODAL'
+        },
+        maxSessionTime: 300,
+        sessionWarning: false
+      },
+      userNotAuthenticated: jest.fn(),
+      handleError: jest.fn(),
+      resetSessionTime: jest.fn(),
+      setSessionWarning: jest.fn(),
+      clearErrorMessage: jest.fn()
+    }
+    const { getByText } = render( 
+      <Router>
+        <UserContext.Provider value={context}>
+          <ModalStatus/>
+        </UserContext.Provider>
+      </Router>);
+
+      expect(getByText('500 Internal Server Error')).toBeInTheDocument();
+      expect(getByText('java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:8011')).toBeInTheDocument();
+
+      await wait(() => {
+        userEvent.click(getByText('OK'));
+      });
+      expect(context.clearErrorMessage).toBeCalledTimes(1);
+  });
+
+  test('Modal can render 500 error and can click Cancel', async () => {
+    const context = {
+      user: {
+        error : { 
+          title: '500 Internal Server Error', 
+          message: 'java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:8011',
+          type: 'MODAL'
+        },
+        maxSessionTime: 300,
+        sessionWarning: false
+      },
+      userNotAuthenticated: jest.fn(),
+      handleError: jest.fn(),
+      resetSessionTime: jest.fn(),
+      setSessionWarning: jest.fn(),
+      clearErrorMessage: jest.fn()
+    }
+    const { getByText, debug } = render( 
+      <Router>
+        <UserContext.Provider value={context}>
+          <ModalStatus/>
+          <NoMatchRedirect/>
+        </UserContext.Provider>
+      </Router>);
+
+      expect(getByText('500 Internal Server Error')).toBeInTheDocument();
+      expect(getByText('java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:8011')).toBeInTheDocument();
+
+      await wait(() => {
+        userEvent.click(getByText('Cancel'));
+      });
+
+      debug()
```

Comment:
can we get rid of this debug from the test.

Reply:
Removed the debug